addons/account/models/account_move.py

```diff
@@ -5091,7 +5091,7 @@ def _mail_track(self, tracked_fields, initial):
     # -------------------------------------------------------------------------
 
     def _prepare_reconciliation_partials(self):
-        ''' Prepare the partials on the current journal items to perform the reconciliation.
+        r''' Prepare the partials on the current journal items to perform the reconciliation.
         /!\ The order of records in self is important because the journal items will be reconciled using this order.
```

Comment:
It seems a bit strange to me to have to use a raw string as docstring just to be able to write /!\\. How about replacing the warning signs with the correct codepoint instead? https://unicode-explorer.com/c/26A0